ethminer/MinerAux.h

```diff
@@ -391,6 +392,10 @@ class MinerCLI
 		{
 			m_api_port = atoi(argv[++i]);
 		}
+		else if ((arg == "--http-port") && i + 1 < argc)
+		{
+			m_http_port = atoi(argv[++i]);
```

Comment:
I' d not use atoi as it does not throw.
Would prefer stoi and implement a check for ports 0-65535 (positive/negative)

Reply:
I was lazy and just cut and paste the lines above.

Reply:
Yes happened to me too. :)

There are also a lot of stol to unsigned which should be stoul.
A good reason to begin amend all of them.

Nevertheless I will review positive to allow me testing.
Thank you jean for this.